pkg/printers/internalversion/describe_test.go

```diff
@@ -2228,6 +2228,7 @@ func TestDescribePodSecurityPolicy(t *testing.T) {
 		"Required Drop Capabilities:\\s*<none>",
 		"Allowed Capabilities:\\s*<none>",
 		"Allowed Volume Types:\\s*<none>",
+		"Allowed Unsafe Sysctls:\\skernel.*,net.ipv4.ip_local_port_range",
```

Comment:
Because these are not just a plain string but regexps, I'd suggest to:
1) replace `\\s` by `\\s*`
2) quote `.` and `*` that are part of the expected output rather than a regexp itself

Reply:
Ah, you're right, I completely missed the dot and the asterisk somehow got left out as well. Thanks!